File: app/src/main/java/com/corphish/nightlight/AboutActivity.java
package com.corphish.nightlight;

import android.support.v4.app.FragmentTransaction;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;

import com.corphish.nightlight.UI.Fragments.AboutFragment;
import com.corphish.nightlight.UI.Fragments.ContributorsFragment;
import com.corphish.nightlight.UI.Fragments.DonateFragment;

public class AboutActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);
        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);

        getSupportActionBar().setDisplayHomeAsUpEnabled(true);

        viewInit();
    }

    private void viewInit() {
        FragmentTransaction fragmentTransaction = getSupportFragmentManager().beginTransaction();
        fragmentTransaction.setCustomAnimations(android.R.anim.slide_in_left, android.R.anim.slide_out_right);

        int containerId = R.id.layout_container;

        fragmentTransaction.add(containerId, new AboutFragment());
        if (getResources().getBoolean(R.bool.contributors_card_enabled)) fragmentTransaction.add(containerId, new ContributorsFragment());
        fragmentTransaction.add(containerId, new DonateFragment());

        fragmentTransaction.commit();
    }
}

File: app/src/main/java/com/corphish/nightlight/AboutActivity.kt
package com.corphish.nightlight

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.corphish.nightlight.design.ThemeUtils

import com.corphish.nightlight.design.fragments.AboutFragment
import com.corphish.nightlight.helpers.ExternalLink
import com.google.android.material.bottomsheet.BottomSheetDialog
import kotlinx.android.synthetic.main.layout_header.*

class AboutActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setTheme(ThemeUtils.getAppTheme(this))
        setContentView(R.layout.activity_about)

        banner_title.text = getString(R.string.about)
        banner_icon.setImageResource(R.drawable.ic_info_24dp)

        if (savedInstanceState == null) viewInit()
    }

    private fun viewInit() {
        val fragmentTransaction = supportFragmentManager.beginTransaction()
        fragmentTransaction.setCustomAnimations(android.R.anim.slide_in_left, android.R.anim.slide_out_right)

        val containerId = R.id.layout_container

        fragmentTransaction.add(containerId, AboutFragment())

        fragmentTransaction.commit()
    }
}
